.github/workflows/release.yml

```
@@ -158,11 +158,57 @@ jobs:
+          mkdir -p pages/.well-known
+          cp .github/security.txt pages/.well-known/
+
+          # Process sitemap.xml with build date
+          sed -i.bak "s/{{BUILD_DATE}}/$BUILD_DATE/g" pages/sitemap.xml
+          sed -i.bak "s/{{VERSION}}/$VERSION/g" pages/sitemap.xml
+          rm -f pages/sitemap.xml.bak
+
+          # Process manifest.json with version
+          sed -i.bak "s/{{VERSION}}/$VERSION/g" pages/manifest.json
+          rm -f pages/manifest.json.bak
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)